go/ias/http/http.go

```diff
@@ -121,9 +143,17 @@ func (e *httpEndpoint) VerifyEvidence(ctx context.Context, evidence *api.Evidenc
 	}
 
 	// Ensure that the AVR is valid.
-	if _, err = ias.DecodeAVR(avr, sig, certChain, e.trustRoots, time.Now()); err != nil {
+	decodedAVR, err := ias.DecodeAVR(avr, sig, certChain, e.trustRoots, time.Now())
+	if err != nil {
 		return nil, fmt.Errorf("ias: failed to parse/validate AVR: %w", err)
 	}
+	if decodedAVR.TCBEvaluationDataNumber < evidence.MinTCBEvaluationDataNumber {
```

Comment:
Why have this check here instead of just returning the AVR and making the caller perform this additional validation (which it needs to do anyway)?